site/gatsby-site/src/components/submissions/SubmissionForm.js

```diff
@@ -291,6 +294,20 @@ const SubmissionForm = () => {
             {...TextInputGroupProps}
           />
         )}
+
+        {isRole('submitter') ? (
```

Comment:
@lmcnulty I believe this PR is ready for merge once the merge conflict is resolved. One request when you resolve the merge conflict,

I was talking with @LKchemposer and Danny this week and we think there is a secondary purpose we would like to use this field for. While we don't want submitting users to provide editor's notes, we do sometimes want to provide the ability for the person submitting to give additional notes that may be helpful to the editor. The editor can then remove the notes or update them to editor's notes when ingesting the report. So here, we can remove the `isRole('submitter')` check, and change the placeholder text to, `Optional context and notes about the incident.`